### PR TITLE
Make missing params public

### PR DESCRIPTION
**Pull request by jason-fox**
Fix related to review comment: https://github.com/telefonicaid/iotagent-node-lib/pull/849#discussion_r446045335

### CODE REVIEW

File: lib/constants.js [hunk 1]
```diff
@@ -73,6 +73,8 @@ module.exports = {
     MONGO_ALARM: 'MONGO-ALARM',
     ORION_ALARM: 'ORION-ALARM',
     IOTAM_ALARM: 'IOTAM-ALARM',
+    ATTRIBUTE_DEFAULT,
+    DATETIME_DEFAULT,
```
Comment (fgalan) [inline lib/constants.js:77]: Should these constants have an actual value (as the others above)?

Comment (fgalan) [reply lib/constants.js:77]: Maybe it's done in L29 and L30 ?

```
const DATETIME_DEFAULT = '1970-01-01T00:00:00.000Z';
const ATTRIBUTE_DEFAULT = ' ';
```

Comment (jason-fox) [reply lib/constants.js:77]: Correct - standard scope inheritance.

Comment (fgalan) [reply lib/constants.js:77]: NTC

**Pull request merged**